helm/fiftyone-teams-app/templates/_helpers.tpl

```diff
@@ -245,6 +245,18 @@ Common Topology Constraints
 {{ end }}
 {{- end }}
 
+{{/*
+Common Init Containers
+*/}}
+{{- define "fiftyone-teams-app.commonInitContainers" -}}
+- name: init-cas
+  image: {{ $.repository }}:{{ $.tag }}
+  command:
+    - 'sh'
+    - '-c'
+    - "until nslookup {{ $.casHostname }}.$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace).svc.cluster.local; do echo waiting for cas; sleep 2; done"
```

Comment:
I hate to be pedantic, but should this be `.casServiceName`?

Reply:
Addressed!